src/neptune_scale/types.py

```diff
@@ -28,3 +35,17 @@ class File:
 
     # Optional destination path in object storage (relative to project namespace).
     destination: Optional[str] = field(default=None)
+
+
+ArrayLike = Union[list[Union[float, int]], "np.ndarray"]
+
+
+@dataclass
+class Histogram:
+    """Represents a histogram with explicit bin edges and counts/densities
+    falling into specific bins.
+    """
+
+    bin_edges: ArrayLike
+    counts: Optional[ArrayLike] = None
+    densities: Optional[ArrayLike] = None
```

Comment:
What kind of values do you pass to `densities`? Does the user need to normalize the counts?